package.json

```diff
@@ -19,10 +19,64 @@
     "auth",
     "authentication"
   ],
-  "author": "SpoonX <info@spoonx.nl>",
-  "license": "MIT",
+  "homepage": "https://github.com/SpoonX/aurelia-acl#readme",
   "bugs": {
     "url": "https://github.com/SpoonX/aurelia-acl/issues"
   },
-  "homepage": "https://github.com/SpoonX/aurelia-acl#readme"
+  "license": "MIT",
+  "author": "SpoonX <info@spoonx.nl>",
+  "devDependencies": {
+    "aurelia-tools": "^0.2.3",
+    "babel": "^6.5.2",
+    "babel-eslint": "^6.0.5",
+    "babel-plugin-syntax-flow": "^6.8.0",
+    "babel-plugin-transform-decorators-legacy": "^1.3.4",
+    "babel-plugin-transform-es2015-modules-amd": "^6.8.0",
+    "babel-plugin-transform-es2015-modules-commonjs": "^6.10.3",
+    "babel-plugin-transform-es2015-modules-systemjs": "^6.9.0",
+    "babel-plugin-transform-flow-strip-types": "^6.8.0",
+    "babel-preset-es2015": "^6.9.0",
+    "babel-preset-es2015-loose": "^7.0.0",
+    "babel-preset-stage-1": "^6.5.0",
+    "conventional-changelog": "1.1.0",
+    "del": "^2.2.1",
+    "gulp": "^3.9.1",
+    "gulp-babel": "^6.1.2",
+    "gulp-bump": "^2.1.0",
+    "gulp-eslint": "^2.0.0",
+    "gulp-yuidoc": "^0.1.2",
+    "isparta": "^4.0.0",
+    "istanbul": "^1.0.0-alpha.2",
+    "jasmine-core": "^2.4.1",
+    "karma": "^0.13.22",
+    "karma-babel-preprocessor": "^6.0.1",
+    "karma-chrome-launcher": "^1.0.1",
+    "karma-coverage": "^1.0.0",
+    "karma-jasmine": "^1.0.2",
+    "karma-jspm": "2.1.1",
+    "object.assign": "^4.0.3",
+    "require-dir": "^0.3.0",
+    "run-sequence": "^1.2.1",
+    "vinyl-paths": "^2.1.0",
+    "yargs": "^4.7.1"
+  },
+  "jspm": {
+    "registry": "npm",
+    "main": "index",
+    "format": "amd",
+    "directories": {
+      "dist": "dist/amd"
+    },
+    "dependencies": {
+      "aurelia-logging": "^1.0.0-rc.1.0.0",
+      "aurelia-pal": "^1.0.0-rc.1.0.0",
+      "aurelia-templating": "^1.0.0-rc.1.0.0"
+    },
+    "devDependencies": {
+      "aurelia-polyfills": "^1.0.0-beta.1.1.0",
+      "babel": "babel-core@^5.8.24",
```

Comment:
those babel/core things can be removed. but it think i rather do it myself later. it's easier :smile: and actually not important at all